components/apimgt/api-store-web/src/site/themes/fancy/templates/api/throttling/template.jag

```diff
@@ -120,10 +120,11 @@ var i, length, app, thumbURL, api = outputs.api, templatesList = outputs.uriTemp
 
 
                                 <% getAuthVal=putAuthVal=postAuthVal=deleteAuthVal=optionAuthVal=""; %>
+                                <% getThrottlingTier=postThrottlingTier=deleteThrottlingTier=optionsThrottlingTier=putThrottlingTier=""; %>
```

Comment:
It's not a best practice to use `undefined`, you can use `null`. but in this case we can't do that since variable is used in other places, and can result in a null pointer or null getting printed to web page.
Best way to avoid this is to not to assign values to variables just in `if`s inside `for`, and set and an appropriate value in `else` as well.